feat(lib): Remove useless mixin rfs and font-size + remove vendor (#3504)
Orange-OpenSource/Orange-Boosted-Bootstrap: site/src/content/docs/getting-started/migration-from-boosted.mdx

- <span class="tag tag-small rounded-none tag-warning">Warning</span> `scss/mixins/_alert.scss` doesn’t exist anymore. If you were importing it in your project, please make sure to remove it.

> - <span class="tag tag-small rounded-none tag-warning">Warning</span> `scss/vendor/_rfs.scss` doesn’t exist anymore. You shouldn’t have used it anyway, but we removed it.

Reviewer: A little less passive aggressive ;)
```suggestion
- <span class="tag tag-small rounded-none tag-warning">Warning</span> `scss/vendor/_rfs.scss` has been removed. It was deprecated and should not have been used.
```

> - <span class="tag tag-small rounded-none tag-negative">Breaking</span> `font-size()` mixin doesn’t exist anymore. You should refer to `get-font-size()` mixin instead.

Reviewer: ```suggestion
- <span class="tag tag-small rounded-none tag-negative">Breaking</span> `font-size()` mixin has been removed. Please refer to `get-font-size()` mixin instead.
```
